src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestAdsAbcdProductsPriority.java

```diff
@@ -45,10 +45,10 @@ public void adsAbcdAdIsDisplayedIfBidersAreDisabledOasis() {
   @Test(groups = {"AdsUAPFamilyAdIsNotDisplayedOnFeaturedVideoPage", "AbcdProductPriorityOasis"})
   public void adsUAPFamilyAdIsNotDisplayedOnFeaturedVideoPage() {
     AdsBaseObject ads = new AdsBaseObject(AdsDataProvider.PAGE_FV.getUrl());
-    verifyAbcdAdIsNotDisplayed(ads);
+    verifyAbcdHIVIAdIsNotDisplayed(ads);
 
-    Assert.assertTrue(ads.slotHasSize(AdsContent.TOP_LB, 728, 90));
-    verifyNoUAPSizesInSlot(ads, AdsContent.TOP_LB);
+    Assert.assertTrue(ads.slotHasSize(AdsContent.HIVI_TOP_LB, 728, 90));
+    verifyNoUAPSizesInSlot(ads, AdsContent.HIVI_TOP_LB);
   }
 
   private void verifyNoUAPSizesInSlot(AdsBaseObject ads, String slot) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyNoUAPSizesInSlot" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)